api/src/main/java/org/apache/iceberg/io/OutputFile.java

```diff
@@ -48,6 +49,7 @@
    *
    * @return an output stream that can report its position
    * @throws RuntimeIOException If the implementation throws an {@link IOException}
+   * @throws AccessControlException If the implementation throws an {@link SecurityException}
```

Comment:
as discussed in the block below, we can just document @throws SecurityException if staging directory creation fails due to missing JVM level permission.

Reply:
I have made the changes accordingly.